LightBlue/Standalone/StandaloneMetadataStore.cs

```diff
@@ -1,51 +1,31 @@
+using Azure.Storage.Blobs.Models;
 using System.Collections.Generic;
 using System.IO;
-
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
+using System.Text.Json;
```

Comment:
Just double checking against the csproj and package.config (now deleted), LightBlue.csproj declares this binary as standard2.0, but standard2.0 doesn't include System.Text.Json since it's only bundled with .net3.0+.
net48 consumers woudl have to pull that package in, but I don't see it listed as a dependency of lightblue.csproj.
Did I miss something?

Reply:
It must be coming in as a transitive dependency somewhere. Is it best practice to explicitly take the dependency?

Reply:
I think if you depend directly on it, it's best to take that dependency explicitly.  The knock on issue is consumers need a way for nuget to pull in stj so LightBlue can run.  This used to be done with the nuspec file, but I think it can be done with csproj now.  The trick I think is to not let consumers propagate the transitive dependency they get from LB so they can pick what they want, while still allowing LB to work.